system-tests/test/component_testing_spec.ts

```diff
@@ -104,6 +104,13 @@ describe(`Angular CLI versions`, () => {
     browser: 'chrome',
     expectedExitCode: 0,
   })
+
+  systemTests.it('angular signals', {
```

Comment:
I think I made a booboo when I merged #29621. I wrote the system test but didn't actually add the code to run it 🙈